doc/install.text

```diff
@@ -0,0 +1,197 @@
+as on Mac OS X.
+The main goal of this quick-start guide is to enable the user to extract and 
+wavelength calibrate an Arc spectrum.
+
+* Install the LSST binary distribution >= 12.0 (https://pipelines.lsst.io/install/conda.html)
+  Here it should not matter whether you install anaconda or miniconda. Please note that
+  there's only a need to install lsst-distrib, not lsst-sims.
+  Don’t forget to setup the LSST environment as mentioned on the website. You will need
+  to do this every time you want to use the pipeline in a new shell, so you might want
+  to add these 2 lines to your .bashrc:
+
+source activate lsst
+source eups-setups.sh
+
+
+* The test data are quite large (~38 MB) and are stored in git using git-lfs. We therefore
+  need to install git-lfs (https://git-lfs.github.com/). Please follow the instructions at
+  https://git-lfs.github.com to installing git-lfs (e.g. using homebrew on os/x), then type
+  
+  git lfs install
+
+  There's no need to issue any "git lfs track" commands
+
+
+* On Linux, all the anaconda packages were compiled with gcc 4.small and are incompatible
+  with modern versions. Therefore, if you are installing the PFS DRP on a Linux machine, 
+  install gcc in anaconda:
+
+conda install gcc
+
+
+* For convenience of this quick-start guide, create a directory for the PFS-DRP repositories:
+
+export PFS_DRP=<your_target_directory>
+(e.g. export PFS_DRP=$HOME/PFS
+mkdir -p $PFS_DRP
+cd $PFS_DRP
+
+
+* Clone the relevant git repositories. Note that currently a newer version of sconsUtils
+  is needed to suppress some irrelevant compiler warnings. Installing and setting up
+  sconsUtils will be obsolete in the next binary LSST Stack installation.
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+git clone git@github.com:lsst/sconsUtils.git
+
+* The next 2 steps are temporary workarounds until fixed:
+* Set $EUPS_PKGROOT
+
+export EUPS_PKGROOT="https://sw.lsstcorp.org/eupspkg/"
+
+
+* Install Eigen with the unsupported modules:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Build and setup the pipeline. Note that setting up drp_stella will overwrite
+  the already set up sconsUtils version with the standard one, so we need to
+  setup sconsUtils again before compiling drp_stella. There are a few warnings
+  when compiling drp_stella, one type of warnings is a doxygen warning "... not
+  found in argument list..." and the other is a SWIG problem ("...destination for
+  this 'memset' call is a pointer..."). Both types of warnings will be silenced 
+  in future version and can be safely ignored.
+
+cd $PFS_DRP/sconsUtils
+git checkout w.2016.34
+setup -r .
+scons -Q opt=3 -j8 --filterWarn
+
+cd $PFS_DRP/obs_pfs
+setup -r .
+scons -Q opt=3 -j8 --filterWarn
+
+cd $PFS_DRP/drp_stella_data
+setup -r .
+
+cd $PFS_DRP/drp_stella
+setup -r .
+setup -r ../sconsUtils
+scons -Q opt=3 -j8 --filterWarn
```

Comment:
I understand you are using `setup -r` because Robert grumped about declaring things. However, if we're doing things this way, we need to explain how a user should set things up the next time they want to use this code. To wit: they will need to do the `setup`s, but not the `scons` commands (and obviously not the cloning, etc, above).